src/instructions/br.rs

```diff
@@ -1,7 +1,5 @@
 use crate::vm::VM;
 
 impl VM {
-    pub fn op_br(&self){
-        
-    }
+    pub fn op_br(&mut self, instr: u16) {}
```

Comment:
There are some clippy errors related to the unused variable instr, but i think it's not bad at all, since its a WIP.